### PR TITLE
[fix](bitmap) fix coredump of bitmap_from_array caused by null array literal

### PR DESCRIPTION
**Pull request by jacktengg**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by jacktengg** [comment]
run buildall

File: be/src/vec/functions/function_bitmap.cpp [hunk 1]
```diff
@@ -382,6 +382,9 @@ class FunctionBitmapAlwaysNull : public IFunction {
             if (check_column<ColumnInt8>(nested_column)) {
                 Impl::template vector<ColumnInt8>(offset_column_data, nested_column,
                                                   nested_null_map, res, null_map);
+            } else if (check_column<ColumnUInt8>(nested_column)) {
```
Comment (xiaokang) [inline be/src/vec/functions/function_bitmap.cpp:385]: Is UInt16/32/64 also needed?

Comment (yiguolei) [reply be/src/vec/functions/function_bitmap.cpp:385]: not need, because doris does not support uint....
Here uint8 is boolean type

**Review by yiguolei — DISMISSED** [review]
LGTM

File: regression-test/suites/query_p0/sql_functions/bitmap_functions/test_bitmap_function.groovy [hunk 2]
```diff
@@ -36,6 +36,9 @@ suite("test_bitmap_function") {
     qt_sql """ select bitmap_to_string(bitmap_from_string("0, 1, 2")) """
     qt_sql """ select bitmap_from_string("-1, 0, 1, 2") """
 
+    // BITMAP_FROM_ARRAY
+    qt_sql """ select bitmap_to_string(BITMAP_FROM_ARRAY([]));"""
```
Comment (yiguolei) [inline regression-test/suites/query_p0/sql_functions/bitmap_functions/test_bitmap_function.groovy:40]: 测试一下新老优化器都可以work

File: be/src/vec/functions/function_bitmap.cpp (hunk 1 above)
Comment (yiguolei) [reply be/src/vec/functions/function_bitmap.cpp:385]: need check datetime it is uint64

Comment (jacktengg) [reply be/src/vec/functions/function_bitmap.cpp:385]: > need check datetime it is uint64

datetime is casted to int64 if used in bitmap_from_array.

**Comment by jacktengg** [comment]
run buildall

**Review by xiaokang — APPROVED** [review]
LGTM

**Review by yiguolei — APPROVED** [review]
LGTM

**Comment by yiguolei** [comment]
run  ut

**Comment by yiguolei** [comment]
run beut

**Comment by yiguolei** [comment]
run feut

**Pull request merged**